plugins/home/src/homePageComponents/FeaturedDocs/FeaturedDocs.tsx

```diff
@@ -0,0 +1,173 @@
+/*
+ * Copyright 2022 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import useAsync from 'react-use/lib/useAsync';
+import {
+  LinkButton,
+  EmptyState,
+  Link,
+  InfoCard,
+  Progress,
+  ErrorPanel,
+} from '@backstage/core-components';
+import { catalogApiRef, CatalogApi } from '@backstage/plugin-catalog-react';
+import { useApi } from '@backstage/core-plugin-api';
+import { EntityFilterQuery } from '@backstage/catalog-client';
+import { ClassNameMap } from '@material-ui/styles';
+
+import { makeStyles, Theme, Typography } from '@material-ui/core';
+
+/**
+ * Props customizing the <FeaturedDocs/> component.
+ *
+ * @public
+ */
+export type FeaturedDocsProps = {
+  /** The entity filter used to display only the intended item/s */
+  filter: EntityFilterQuery;
+  /** An optional color which can be customized through themes */
+  color?: 'inherit' | 'primary' | 'secondary';
```

Comment:
Since you can define a primary and secondary color in your global theme, the thought is to allow users to simply specify which color the links should be if not the default `primary`.  
The argument would be that this approach would be a cleaner way to define a color since the hex would only exist in the global theme, and not additionally in the `customStyles` or in the `BackstageFeaturedDocs` override. 

But I can remove this prop since we want people to customize colors using `BackstageFeaturedDocs` if needed